pkg/hns/netconf_windows.go

```diff
@@ -45,6 +54,37 @@ type policy struct {
 	Value json.RawMessage `json:"value"`
 }
 
+func GetDefaultDestinationPrefix(ip *net.IP) string {
+	destinationPrefix := "0.0.0.0/0"
+	if ipv6 := ip.To4(); ipv6 == nil {
+		destinationPrefix = "::/0"
+	}
+	return destinationPrefix
+}
+
+func (n *NetConf) applyLoopbackDSR(ip *net.IP) {
+	value := fmt.Sprintf(`"Destinations" : ["%s"]`, ip.String())
```

Comment:
Please use `%q` instead of `"%s"`.

Reply:
hmm, probably not necessary in this case since it's just building json.